rastervision_pytorch_learner/rastervision/pytorch_learner/learner_config.py

```diff
@@ -210,6 +211,19 @@ def validate_config(self):
                 'class_loss_weights is not supported with external_loss_def.')
 
 
+@register_config('plot_options')
+class PlotOptions(Config):
+    """Config related to plotting."""
+    transform: Optional[dict] = Field(
+        None,
+        description='An Albumentations transform serialized as a dict that '
```

Comment:
I would just be a little more verbose and add that the main purpose of this is to undo any augmentations that you don't want to be included in the plot, eg. normalization.